test/restoreEstudio.js

```diff
@@ -0,0 +1,196 @@
+    chai.should();
+    chai.use(chaiAsPromised);
+});
+
+
+describe('Create Estudio test', function () {
+  /**
+  * Integration test suite for testing the Login.
+  *
+  * Test if Login works correctly in the app.
+  *
+  * Attributes
+  * ----------
+  * application: spectron
+  *   Driver to launch the application.
+  */
+
+  //The time out to launch the app in the test.
+  this.timeout(10000);
+
+  let totalEstudios;
+  let estudioId;
+
+  before(() => {
+    //connect to db
+    connect(dbUri).then((db) => {
+      database = db;
+      return Estudio.count()
+    })
+    .then((total) => {
+      totalEstudios = total;
+      let e = Estudio.create({
+        //change later for config
+        tokenCapturista: config.apiToken,
+        familia: {
+          bastardos: 10,
+          estadoCivil: 'Soltero',
+          calle: 'Erizo',
+          colonia: 'Fs',
+          codigoPostal: 76150,
+          localidad: 'Otro'
+        },
+        status: 'Eliminado',
+      });
+      return e.save();
+    })
+    .then((newEstudio) => {
+      estudioId = newEstudio._id;
+    })
+    // end Database connection
+  });
+
+  // Before everything we launch the app.
+  beforeEach(async function () {
+    // Launch the application
+    this.app = new Application({ path: electronPath, args:['.'] });
+    return this.app.start();
+  });
+
+
+  // After test is complete we stop the app.
+  afterEach(async function () {
+    if (this.app && this.app.isRunning()) {
+      return this.app.stop();
+    }
+  });
+
+  after(() => {
+    //delete database
+    return database.dropDatabase();
+  });
+  /**
+  * Test Restore estudio button with estudio._id
+  *
+  * Test if the restore-estudio button exists in the
+  * application.
+  */
+  it('should see restore estudio button', function () {
+    const client = this.app.client;
+    return client.setValue('#username',config.username)
+      .setValue('#password', config.password)
+      .click('#submit-login')
+      .click('#show-deleted')
+      .waitForVisible('#home-btn')
+      .then(() => {
+        return client.$('#restore-estudio-' + estudioId);
+      })
+      .then(async (restoreBtn) => {
+        assert.isNotNull(restoreBtn.value);
+      });
+  });
+  // /**
+  // * Test delete estudio
+  // *
+  // * This test will check if a modal shows before deleting an estudio
+  // */
+  // it('should see modal before deleting an estudio', async function () {
+  //   const client = this.app.client;
+  //   return client.setValue('#username',config.username)
+  //     .setValue('#password', config.password)
+  //     .click('#submit-login')
+  //     .click('#delete-estudio-' + estudioId)
+  //     .then(async () => {
+  //       await sleep(1000);
+  //       return client.getText('#modalContentId');
+  //     })
+  //     .then((modalText) => {
+  //       assert.equal(modalText, 'No podras recuperar el estudio después de esta acción.');
+  //     });
+  // });
+  // /**
+  // * Test delete estudio
```

Comment:
@erikiado a comment of a test that will no be used, if you look at all the PR you will see that it is used later.